server.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const bodyParser = require('body-parser');
+const db = require('./db');
+const router = require('./routes/router')
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Do we need this? I say no.

This backend exposes its api as RESTful JSON.  We are not expecting actual form requests since, I'm assuming, we'll be making AJAX requests from a React frontend that updates it's own state, not rerequesting+refreshing the browser.

Single page app vs. traditional webpages that refresh each request.

Reply:
ahhh i see. That makes sense. React will handle state for us. Done.